lib/health_cards/verifier.rb

```diff
@@ -4,11 +4,35 @@ module HealthCards
   # Verifiers can validate HealthCards using public keys
   class Verifier
     attr_reader :keys
+    attr_accessor :resolve_keys
+
+    # Verify a HealthCard
+    #
+    # This method _always_ uses key resolution and does not depend on any cached keys
+    #
+    # @param verifiable [HealthCards::JWS, String] the health card to verify
+    # @return [Boolean]
+    def self.verify(verifiable)
+      jws = JWS.from_jws(verifiable)
+      key_set = resolve_key(jws)
+      key = key_set.find_key(jws.kid)
+      jws.public_key = key
```

Comment:
This code seems at least somewhat duplicative of the instance method

Reply:
I created a `verify_using_key_set` method to hold the shared logic.

First (This commit: https://github.com/dvci/health_cards/pull/39/commits/7c478f65f9a34879a7ed1df46d196858a6058fbd)  I just added this as a class method that the instance `verify` method could also call

Then (This commit: https://github.com/dvci/health_cards/pull/39/commits/1b1b4722dd00df3fed7a520fc021b59da15280ee) I moved `verify_using_key_set` and `resolve_key` to a new module so that I could just `include`/`extend` `Verifier` with it (seemed a little cleaner)

Do you prefer one approach over another or have a different suggestion?